### DISCUSSION TITLE
Why do not merged DynamicFieldsModelSerializer ?

### DISCUSSION THREAD

**Discussion by KimSoungRyoul** [encode/django-rest-framework#7972] [Question & Answer]
https://www.django-rest-framework.org/api-guide/serializers/#dynamically-modifying-fields

I think this is highly versatile feature. 

not only refer to document but also It seems more appropriate to add it as a new feature

or  more  efficient way 
~~~Python
# example: more  efficient DynamicSerializer

class StaffSerializer(ModelSerializer):
     boss  = DynamicSerializer(model=Staff, field=("id", "name"), required=True, ....)
     task_set = DynamicSerializer(
                  model=Task, field=("id","task",....), many=True, extra_kwargs={"task":{"requried": True}},....
     )


~~~

I want to hear your opinion.

thanks

**Comment by jerinpetergeorge** [comment]
Just curious, What about a _**nested dynamic serializer**_?

**Reply by KimSoungRyoul** [reply]
it's more intuitive name.  lol~

**Comment by KimSoungRyoul** [comment]
if this feature doesn't  harm the philosophy of DRF, I want to PullRequest `DynamicSerializer ` or `NestedDynamicSerializer`

**Comment by lovelydinosaur** [comment]
We're pretty much treating REST framework as feature complete at this point. We're not really accepting new functionality unless it's to do with keeping up to date with newer version of Django, or filling in blanks in our schema generation.
